src/main/java/fi/hsl/transitdata/vehicleposition/application/gtfsrt/GtfsRtOccupancyStatusHelper.java

```diff
@@ -42,102 +41,27 @@ public GtfsRtOccupancyStatusHelper(NavigableMap<Integer, GtfsRealtime.VehiclePos
     }
 
     public Optional<GtfsRealtime.VehiclePosition.OccupancyStatus> getOccupancyStatus(Hfp.Payload hfpPayload, PassengerCount.Payload passengerCountPayload) {
-        
-        boolean containsId = false;
-        
-        if (passengerCountEnabledVehicles != null) {
-            if (hfpPayload == null) {
-                throw new RuntimeException("hfpPayload is null");
-            }
-    
-            int oper = hfpPayload.getOper();
-            int veh = hfpPayload.getVeh();
-            
-            try {
-                containsId = passengerCountEnabledVehicles.contains(oper + "/" + veh);
-            } catch (Exception x) {
-                log.error("Contains failed (oper=" + oper + ", veh=" + veh + ")");
-                throw new RuntimeException("Contains failed (oper=" + oper + ", veh=" + veh + ")", x);
-            }
-        }
-        
-        if (passengerCountEnabledVehicles == null || containsId) {
+        if (passengerCountEnabledVehicles == null || passengerCountEnabledVehicles.contains(hfpPayload.getOper() + "/" + hfpPayload.getVeh())) {
             //If occu field is 100, the driver has marked the vehicle as full
-            try {
-                if (hfpPayload.getOccu() == 100) {
-                    return Optional.of(GtfsRealtime.VehiclePosition.OccupancyStatus.FULL);
-                }
-            } catch (Exception x) {
-                log.error("GetOccu 1 failed");
-                throw new RuntimeException("GetOccu 1 failed", x);
+            if (hfpPayload.getOccu() == 100) {
```

Comment:
I wonder if it would make sense to allow values higher than 100 here.
```suggestion
            if (hfpPayload.getOccu() >= 100) {
```

Reply:
Is this absolute number or procent?